src/include/daos/sys_debug.h

```diff
@@ -10,9 +10,7 @@
 #ifndef __DC_DEBUG_H__
 #define __DC_DEBUG_H__
 
-#include <daos/common.h>
-#include <daos/tse.h>
-#include <daos_types.h>
+#include <daos_task.h>
```

Comment:
so why here add daos_task.h but other places like https://github.com/daos-stack/daos/pull/13613/files#diff-930d69e66aac2e78ef8537ea29d976fc69620cbdeee43d117662b9e6f7f1f036R14 you added aos/tse.h

seems inconsistent.